Multisig/UI/Dapps/DappsViewController.swift

```diff
@@ -26,7 +27,7 @@ class DappsViewController: UIViewController, UITableViewDataSource, UITableViewD
         case dapp(String)
 
         enum WalletConnect: SectionItem {
-            case activeSession(WCSession)
+            case activeSession(Any)
```

Comment:
Would it make sense to have a protocol that both session types V1 & V2 implement?

Reply:
I like that, will try to do

Reply:
I will do in future pr